sys/ansible/playbooks/rollback.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: "Rollback badge-poser"
+  hosts: "web-1"
+  become: true
+
+  # TASKS
+  # ############################################################################
+
+  tasks:
+    # ROLLBACK
+    # ########
+
+    - name: "Retrieve last folder"
+      shell: ls -tr /application/releases/ | tail -n 2 | head -n 1
+      register: previous_release
+
+    - pause:
+        prompt: "Do you want to revert to the version {{ previous_release }}?"
+
+    - name: "Release"
+      deploy_helper:
+        path: /application
+        release: "{{ previous_release }}"
+        state: finalize
+        clean: no
+      notify: "reload web services"
+
+    - name: "Check cachetool.phar already exists"
+      stat: path="{{deploy_helper.shared_path}}/cachetool.phar"
+      register: stat_cachetool_result
+
+    - name: "Download cachetool.phar"
+      get_url:
+        url: "https://gordalina.github.io/cachetool/downloads/cachetool-3.2.1.phar"
+        dest: "{{deploy_helper.shared_path}}/cachetool.phar"
+        mode: '+x'
+      when: not stat_cachetool_result.stat.exists
+
+    - name: "Cachetool Opcache Reset"
+      shell: "php {{deploy_helper.shared_path}}/cachetool.phar opcache:reset --fcgi=/run/php-fpm/www.sock"
+      listen: "cachetool_opcache_reset"
```

Comment:
where is the trigger of `cachetool_opcache_reset `?

Reply:
fixed